gnucash/python/init.py

```diff
@@ -20,15 +24,17 @@
 # Restore the SIGTTOU handler
 signal.signal(signal.SIGTTOU, old_sigttou)
 
+# output debug information if gnucash has been started with
+# gnucash --debug --extra
 if gnc_prefs_is_extra_enabled() and gnc_prefs_is_debugging_enabled():
     print("Hello from python!")
-    print("test", sys.modules.keys())
-    print("test2", dir(_sw_app_utils))
+    print("\nsys.modules.keys(): ", sys.modules.keys())
+    print("\ndir(_sw_app_utils): ", dir(_sw_app_utils))
```

Comment:
How about adding the newlines at the end of the print statements rather than at the beginning ? To my eye that is easier to read.

Reply:
ok